proxies/live/apiproxy/policies/Quota.General.xml

```diff
@@ -0,0 +1,7 @@
+<Quota name="Quota.General" type="rollingwindow">
+    <Interval>8</Interval>
+    <TimeUnit>hour</TimeUnit>
+    <Allow countRef="apiproduct.requestLimit_general"/>
+    <Distributed>true</Distributed>
+    <Synchronous>false</Synchronous>
```

Comment:
The rate limiting shared flow uses <Synchronous>true</Synchronous>. 
Not sure how PR 290 is going to turn out, but if we end up using the shared flow, there will be a minor inconsistency here. Might not be significant though...
From https://docs.apigee.com/api-platform/reference/policies/quota-policy#synchronous looks like setting it to true can result in a few throughput issues.

Reply:
I did originally have `synchronized` defined as `true` but we decided we were happy with clients being able to send a few more requests than they should with the SpikeArrest policy also being in place.